Fixes

.../gcc/reload1.c:89:24: error: use of an operand of type 'bool' in 'operator++' is forbidden in C++17

Fixed upstream in GCC6+ in d57c99458933.

---
 gcc/reload.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/gcc/reload.h
+++ b/gcc/reload.h
@@ -166,7 +166,7 @@
      value indicates the level of indirect addressing supported, e.g., two
      means that (MEM (MEM (REG n))) is also valid if (REG n) does not get
      a hard register.  */
-  bool x_spill_indirect_levels;
+  unsigned char x_spill_indirect_levels;
 
   /* True if caller-save has been reinitialized.  */
   bool x_caller_save_initialized_p;
